chapter_03_unit_test_first_view.asciidoc

```diff
@@ -14,7 +14,7 @@ We finished the last chapter with a functional test failing,
 telling us that it wanted the home page for our site to have ``To-Do'' in its title.
 It's time to start working on our application.
 In this chapter, we'll build our first HTML page, find out about URL handling,
-and creating responses to HTTP requests with Django's view functions.
+and create responses to HTTP requests with Django's view functions.
```

Comment:
i think this one works either way - my way was "we'll find out about (a) url handling and (b) creating responses"

Reply:
I agree that both ways work well. For that way I would propose
```
In this chapter, we'll build our first HTML page and find out about (a) URL handling and (b) creating responses...
```
i.e. remove the comma between 'page' and 'find'.